Mindee Invoice.ipynb

```diff
@@ -0,0 +1,417 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "sufficient-charles",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mindee import Client\n",
+    "import json\n",
+    "import cv2\n",
+    "from matplotlib import pyplot as plt\n",
+    "import numpy as np\n",
+    "import math\n",
+    "from IPython.lib.pretty import pretty\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "chicken-still",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mindee_client = Client(\n",
+    "    invoice_token= '<insert api key>',\n",
+    "    raise_on_error=True\n",
```

Comment:
I'd suggest adding a comment or even a text cell with the link / explanation on how to get your API key

Reply:
in the readme?  Is that enough?

Reply:
Up to you: if people end up opening your notebook on Google Colab (not on their actual computer), they might skip the reading from the README :man_shrugging:

Reply:
fixed :)